src/buildExec.ts

```diff
@@ -78,18 +78,21 @@ const buildCommitExec = async (): Promise<{
 }> => {
   const commitParent = core.getInput('commit_parent');
   const gitService = getGitService();
-  const overrideBranch = core.getInput('override_branch');
+  let overrideBranch = core.getInput('override_branch');
   const overrideCommit = core.getInput('override_commit');
   const overridePr = core.getInput('override_pr');
   const slug = core.getInput('slug');
   const token = await getToken();
+  if (token == null) {
+    overrideBranch = context.payload.pull_request?.head.label;
```

Comment:
what happens if a token is null AND there is an overrideBranch specified?

Reply:
it gets overwritten